tests/integration_tests/test_client_creation.py

```
@@ -0,0 +1,257 @@
+        print(f"\n✅ collection.peek() with data returned {len(preview_with_data['ids'])} items")
+        
+        # Test 16: collection.peek() with different limit
+        preview_all = collection.peek(limit=10)
+        assert len(preview_all["ids"]) == 3  # All 3 items
+        print(f"\n✅ collection.peek(limit=10) returned {len(preview_all['ids'])} items")
+        
+        # Clean up: delete the test collection table
+        try:
+            db_client._server._execute(f"DROP TABLE IF EXISTS `{table_name}`")
+            print(f"   Cleaned up test table: {table_name}")
+        except Exception as cleanup_error:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)